Directory.Build.props

```diff
@@ -8,8 +8,7 @@
     <OldestSupportedDotNetVersion>net8.0</OldestSupportedDotNetVersion>
     <NetCoreVersions>$(LatestSupportedDotNetVersion)</NetCoreVersions>
     <NetCoreVersions Condition="'$(LatestSupportedDotNetVersion)' != '$(OldestSupportedDotNetVersion)'">$(LatestSupportedDotNetVersion);$(OldestSupportedDotNetVersion)</NetCoreVersions>
-    <NetStandardVersions>netstandard2.0</NetStandardVersions>
```

Comment:
Please also remove the Condition Variables property group below now too.
IsNetStandard and IsNetFramework are no longer relevant and IsNetCore is the only remaining condition so it is also redundant.